tests/ast/data/functions.test

```diff
@@ -1155,7 +1155,7 @@ res277 = res112.select(when(col("a") % 2 == 0, lit("even")))
 
 res278 = res112.select(iff(col("a") % 2 == 0, lit("even"), lit("odd")))
 
-res279 = res112.select(in_(Empty$, Empty$), in_([col("A"), col("B"), col("A")], Empty$), in_([col("col1"), col("col2")], [[1, "a"], [2, "b"]]))
+res279 = res112.select(in_(Add a handler for Empty$ in Unparser.expr(), Add a handler for Empty$ in Unparser.expr()), in_([col("A"), col("B"), col("A")], Add a handler for Empty$ in Unparser.expr()), in_([col("col1"), col("col2")], [[1, "a"], [2, "b"]]))
```

Comment:
I presume this is intentional, since this matches the current implementation. Is that correct?

Reply:
Yes. To be clear to everyone else who might be looking at this, I changed the catch-all handler (which we expect to assert when we're done) to generate an extended string instead of just the name of the AST entity that isn't being handled. It's clearly beyond the scope of this PR to fix the underlying issue.